File: Python/1020.py
# BEE 1020 | IDADE EM DIAS

V = int(input())
anos = V // 365
V %= 365
meses = V // 30
V %= 30

print("%d ano(s)" % anos)
print("%d mes(es)" % meses)
print("%d dia(s)" % V)

File: PHP/1020.php
<?php

# BEE 1020 | AGE IN DAYS

$daysT = (int) fgets(STDIN);

$years = $daysT / 365;
$daysT %= 365;
$months = $daysT / 30;
$days = $daysT % 30;

echo (int) $years . " ano(s)". "\n";
echo (int) $months . " mes(es)". "\n";
echo (int) $days . " dia(s)". "\n";